GoodThingsComein3s/Views/RestaurantTableViewCell.h

```diff
@@ -7,12 +7,13 @@
 
 #import <UIKit/UIKit.h>
 #import "Restaurant.h"
+#import <Parse/Parse.h>
 
 NS_ASSUME_NONNULL_BEGIN
 
 @protocol RestaurantTableViewCellDelegate
 
--(void) userLoginSignUp;
+-(void) addLikedRestaurantToUser:(PFUser *)currUser restaurant:(Restaurant *)restaurant;
```

Comment:
Same feedback that the delegate should only be notified of actions occurring and not what to do when those actions occur. So this can be renamed to something like:

`- (void)restaurantTableViewCell:(RestaurantTableViewCell *)restaurantCell didTapLikeForRestaurant:(Restaurant *)restaurant;`

Removing the currUser param because again, [PFUser currentUser] can be used to get the current user from anywhere in the app